terraform/service_generate.tf

```diff
@@ -0,0 +1,148 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+# Create and deploy the service
+#
+
+resource "google_service_account" "generate" {
```

Comment:
I kinda wonder if we should always build and deploy this service, but instead have the default Cloud Scheduler settings be "never" (default of February 31st or something). That would technically add $0.10, but it would reduce the variability in these configs (and the conditional on count).

Reply:
yeah - I think always deploy it but not schedule it is good.

Reply:
Done. Please re-review @mikehelmick